ISISComputingGroup/system_tests
# Ticket 5430: Add datastreaming system tests

### Description of work

Added datastreaming system tests based on those from the [filewriter](https://github.com/ess-dmsc/kafka-to-nexus). 

Note that once this is merged we need to:
* Change the windows 10 system test to point at the master branch
* Change the other system tests to provide a parameter for which machine to run on

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/5430.

To test:
* Install and start docker for windows
* With https://github.com/ISISNeutronMuon/ISISICP/pull/16 run the datastreaming tests and confirm they pass

### Documentation
Updated docs at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Datastreaming

---

#### Code Review


### Final Steps
